tests/migrations/test_state.py

```diff
@@ -870,6 +871,38 @@ class Meta:
         self.assertEqual([name for name, mgr in food_state.managers], ['food_mgr'])
         self.assertEqual(food_state.managers[0][1].args, ('a', 'b', 1, 2))
 
+    def test_metaclass_conflict(self):
+        """
+        Test making class with complex metadata inheritance
+        """
+        new_apps = Apps(['migrations'])
+
+        class DummyMetaClass(type):
+            pass
+
+        class Mixin(six.with_metaclass(DummyMetaClass, object)):
+            mixin_attr = True
+
+        class ItermidiateMetaClass(ModelBase, DummyMetaClass):
+            pass
+
+        # create the same Model without metaclassmaker as you would normally do
+        class MetaclassTestModel(six.with_metaclass(ItermidiateMetaClass, models.Model, Mixin)):
+            class Meta:
+                app_label = "migrations"
+                apps = new_apps
+                swappable = 'TEST_SWAPPABLE_MODEL'
+
+        model_state = ModelState('migrations', 'migrations.test_state.MetaclassTestModel',
+                                 [], bases=[models.Model, Mixin])
+        try:
+            class_obj = model_state.render(new_apps)
+        except TypeError:
+            class_obj = None
+
+        self.assertIsNotNone(class_obj, "Not resolved metaclass conflict")
```

Comment:
I can't, this is negation of what `assertRaisesMessage` is doing and there is no `assertNotRaisesMessage`, if you want I will write `asserNotRaisesMessage`, or I can just rise AssertError after catching TypeError. Which would be better?